<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MovementsViewerWidget</class>
 <widget class="QWidget" name="MovementsViewerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Movements</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="quitButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Quit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="validButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Valid the bill</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="valAndRecButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Valid and record</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="recordButton">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Record only</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="valueLabel">
       <property name="text">
        <string>Value</string>
       </property>
       <property name="buddy">
        <cstring>valueDoubleSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="valueDoubleSpinBox"/>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="percentLabel">
       <property name="text">
        <string>Percentage professionnal</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="buddy">
        <cstring>percentDoubleSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QDoubleSpinBox" name="percentDoubleSpinBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="movementsLabel">
       <property name="text">
        <string>Movements</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="movementsComboBox">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="dateLabel">
       <property name="text">
        <string>Date</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="buddy">
        <cstring>dateEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QDateEdit" name="dateEdit">
       <property name="displayFormat">
        <string notr="true">dd/MM/yy</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="bankLabel">
       <property name="text">
        <string>bank</string>
       </property>
       <property name="buddy">
        <cstring>bankComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="bankComboBox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="yearLabel">
       <property name="text">
        <string>Year</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QComboBox" name="yearComboBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="detailsLabel">
       <property name="text">
        <string>Details</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="buddy">
        <cstring>detailsEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QLineEdit" name="detailsEdit">
       <property name="minimumSize">
        <size>
         <width>240</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>valueDoubleSpinBox</tabstop>
  <tabstop>percentDoubleSpinBox</tabstop>
  <tabstop>detailsEdit</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>bankComboBox</tabstop>
  <tabstop>validButton</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
